ios/FluentUI/Core/ControlHostingView.swift

```diff
@@ -56,6 +56,11 @@ open class ControlHostingView: UIView {
         addSubview(hostedView)
         hostedView.translatesAutoresizingMaskIntoConstraints = false
 
+        // Initialize our frame with the hosted view's initial bounds so we don't start as a (0,0,0,0) view.
+        // Future updates will be handled by the autolayout constraints below.
+        hostedView.sizeToFit()
+        self.frame = hostedView.bounds
```

Comment:
Nit: seems like this is more about initializing the hosting view than configuring the hosted view, so maybe the call to set the frame should be in the init instead? Also, I can't leave a comment up there very easily through GitHub, but is there any chance of the intrinsic content size getting out of sync?

Reply:
> maybe the call to set the frame should be in the init instead?

I originally decided to lump all stuff about `hostingController.hostedView` view sizing and hierarchy in our `init` method, after calling `super.init()`. But that bloated the method quite a bit, so I intentionally broke that code out into this separate helper method. It's only called during `init`, so the effect is similar.

> is there any chance of the intrinsic content size getting out of sync?

Hm I don't believe so--the `intrinsicContentSize` is a generated var, so it will always recalculate whenever it's requested.